internal/handlers/tracer/tracer.go

```diff
@@ -317,6 +302,33 @@ EXIT:
 	return nil
 }
 
+func updateAllowedIPMaps(allowedIPMap *ebpf.Map, arg *domain.Data) error {
+	for _, ipstr := range arg.AllowedIPs {
+		ip := ipstr.To4()
+		var ipUint32 uint32
+		if len(ip) > 16 {
+			ipUint32 = binary.LittleEndian.Uint32(ip[12:16])
+		} else {
+			ipUint32 = binary.LittleEndian.Uint32(ip)
+
+		}
```

Comment:
A redundant new line can be removed.

```suggestion
			ipUint32 = binary.LittleEndian.Uint32(ip)
		}
```